src/main/java/net/sf/jabref/exporter/SaveDatabaseAction.java

```diff
@@ -288,7 +288,10 @@ private boolean saveDatabase(File file, boolean selectedOnly, String encoding) t
             String tryDiff = Localization.lang("Try different encoding");
             int answer = JOptionPane.showOptionDialog(frame, builder.getPanel(), Localization.lang("Save database"),
                     JOptionPane.YES_NO_CANCEL_OPTION, JOptionPane.WARNING_MESSAGE, null,
-                    new String[] {Localization.lang("Save"), tryDiff, Localization.lang("Cancel")}, tryDiff);
+                    // @formatter:off
+                    new String[] {Localization.lang("Save"), tryDiff,
+                            Localization.lang("Cancel")}, tryDiff);
+                    // @formatter:on
```

Comment:
I think, the current `syncLang.py` can handle a single `Localization.lang` call per line. Therefore, the split among different lines is required.

We could use a [Java parser written in Python](https://github.com/musiKk/plyj/tree/java8) to improve `syncLang.py`, but I'm not sure, it's worth the effort.